conf/themes/silverstripe/layout/layout.twig

```diff
@@ -7,7 +7,7 @@
 
 	{# CUSTOM: Toolbar support #}
 	<link rel="stylesheet" type="text/css" href="//www.silverstripe.org/themes/ssv3/css/toolbar.min.css" />
-
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```

Comment:
```suggestion
    <script src="{{ path('js/jquery-1.11.1.min.js') }}"></script>
```

https://github.com/FriendsOfPHP/Sami/blob/master/Sami/Resources/themes/default/layout/base.twig#L12

But do not bother to work on this, I am planning to migrate to Doctum